CollAction/Controllers/ProjectsController.cs

```diff
@@ -412,13 +413,31 @@ public ViewResult StartInfo()
 
             if (success)
             {
+                var projUrl = this.Url.Action("Details", "Projects", new { name=commitProjectViewModel.ProjectName}, HttpContext.Request.Scheme); 
```

Comment:
If the typename isn't long, it's probably best to not use "var". Also, best just write out the variable name, ie, projectUrl.

Reply:
fixed